main.py

```diff
@@ -1,10 +1,48 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+
+    with open("players.json", "r") as file:
```

Comment:
flag "r" set as default, you can don't use them, if you want just to read file information